ert3/engine/_run.py

```diff
@@ -48,17 +48,6 @@ def _load_input_records(ensemble, workspace_root, experiment_name):
     return records
 
 
-def _create_forward_model(stages_config, ensemble):
```

Comment:
As it is now the `evaluator` that extracts the forward model, the asserts have been moved into that sub module.